resources/views/errors/status.blade.php

```diff
@@ -0,0 +1,11 @@
+@if (Session::has('status'))
+    <div>{{ Session::get('status') }}</div>
+@endif
+
+@if(isset($errors) && $errors->any())
```

Comment:
The errors middleware will always add this to the view, so you don't need the `isset()` check here.